### ISSUE TITLE
folder too big for server?

### ISSUE THREAD

**Issue by fredphoesh** [thunderbird/import-export-tools-ng#501]
Hello all,
I'm trying to move gmails from a work gmail account to my personal one, and of the 16000 messages in my inbox, only 3218 will copy when I drag drop the Local Folder. I get this error:
![toobig](https://github.com/thunderbird/import-export-tools-ng/assets/5565996/d3b64c48-0e1b-4ecc-ba89-c014d2060de2)

[Here is a video ](https://videoblurb.com/mp/importexporttools.mp4)showing my process of drag dropping. I think the message did not pop up here because I had Thunderbird in that corner of the desktop. Regardless, you can see the problem, ie files not copying.

Any ideas why this is, and is there a work-around?
Thanks,
Mark.
Thunderbird 115.4.3
ImportExportTools Version 14.0.1

**Comment by cleidigh** [comment]
Hi Mark
I watched the video so I see clearly what you are trying to do. Thanks, wish many users would do that I can't being disabled

So you are trying to copy/move large folders from your Local Folders to one of your IMAP accounts, 

The first thing I would like to explain is that ImportExportTools is NOT involved in this operation at all, you are performing a pure Thunderbird operation. So nothing IETNG can do about it.

That being said let me tell you what I believe is happening. What your operation is requesting is to perform a very large number of IMAP message uploads. This puts a big burden on Thunderbird, your mail server and Internet connection.

Now from what you have shown me, it is the server giving the error, Thunderbird is just displaying it. So I see two scenarios.
- The server is overwhelmed with requests and the message is not really clear
- There are one or more messages you are uploading that exceed the max message size for the server.

So what to do. Basically I suggest you copy selected sets of messages and see where it fails. Then see if there are specific messages that fail.
@cleidigh

**Comment by fredphoesh** [comment]
Hi Christopher, it's very kind of you spending the time to give advice when it's not an issue with this addon. You are quite right. I found there was ONE message out of all those thousands which was a thread of about 26 messages, and many of them had an attachment. Perhaps google was seeing them as all part of one message, I don't know, so I deleted the unneeded attachments on TB and got the whole job done. Thanks again! Mark.
